server/index.js

```diff
@@ -1,12 +1,68 @@
 const express = require('express');
+const sampleData = require('../database/sampleDatas/imageServiceSample.js')
 
 
 const app = express();
 
 app.use(express.static('public'));
+app.use(express.json());
+app.use(express.urlencoded())
 
 //console.log(__dirname + '/public/lib')
 
-var port = 3000;
+app.get('/:productId', (req, res) => {
+  var product_id = parseInt(req.params.productId);
+  var data = sampleData.find(data => data.product_id === product_id);

```

Comment:
Consider using destructuring to reduce the amount of typing needed later.

.e.g. 
`let { carousel_image, thumbnail, cover_image, description_image, description_gifs } = sampleData.find(data => data.product_id === product_id);`

That will eliminate needing to data `data.thumbnail`, `data.cover_image`, etc. and just use `thumbnail`, `cover_image`